t/v3/health_check.t

```diff
@@ -157,19 +157,54 @@ failed to get ngx.shared dict: error_shm_name
             })
 
             local res, err = etcd:set("/trigger_unhealthy", { a='abc'})
-            ngx.say("done")
+            ngx.say(err)
         }
     }
 --- request
 GET /t
 --- error_log eval
 qr/update endpoint: http:\/\/127.0.0.1:42379 to unhealthy/
 --- response_body
-done
+http://127.0.0.1:42379: connection refused
+
+
+
+=== TEST 5: trigger unhealthy with watch
+--- http_config eval: $::HttpConfig
+--- config
+    location /t {
+        content_by_lua_block {
+            local health_check, err = require "resty.etcd.health_check" .init({
+                shm_name = "etcd_cluster_health_check",
+                fail_timeout = 10,
+                max_fails = 1,
+            })
+
+            local etcd, err = require "resty.etcd" .new({
+                protocol = "v3",
+                http_host = {
+                    "http://127.0.0.1:42379",
+                    "http://127.0.0.1:22379",
+                    "http://127.0.0.1:32379",
+                },
```

Comment:
not using user/password here so auth would not be triggered, thus `request_chunk` would not visit `request_uri` and get error from it before its own expected error